File: src/main/java/com/gtladd/gtladditions/api/machine/IWirelessElectricMultiblockMachine.java
package com.gtladd.gtladditions.api.machine;

import com.gtladd.gtladditions.api.machine.trait.IWirelessNetworkEnergyHandler;
import org.jetbrains.annotations.Nullable;

public interface IWirelessElectricMultiblockMachine {

    @Nullable
    default IWirelessNetworkEnergyHandler getWirelessNetworkEnergyHandler() {
        return null;
    }

    default void setWirelessNetworkEnergyHandler(IWirelessNetworkEnergyHandler trait) {}
}

File: src/main/java/com/gtladd/gtladditions/api/machine/IWirelessElectricMultiblockMachine.kt
package com.gtladd.gtladditions.api.machine

import com.gtladd.gtladditions.api.machine.trait.IWirelessNetworkEnergyHandler

interface IWirelessElectricMultiblockMachine {
    fun getWirelessNetworkEnergyHandler(): IWirelessNetworkEnergyHandler? = null

    fun setWirelessNetworkEnergyHandler(trait: IWirelessNetworkEnergyHandler) {}
}